src/components/Form/InputWrapper.tsx

```diff
@@ -18,11 +18,13 @@ function computeComponentSpecificRegistrationParams<TInput extends ValidInputs>(
     readonly shouldSetTouchedOnBlurOnly: boolean,
 } {
     if (InputComponent === TextInput) {
-        const isEffectivelyMultiline = Boolean(multiline ?? autoGrowHeight);
+        // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+        const isEffectivelyMultiline = Boolean(multiline || autoGrowHeight);
```

Comment:
You're right; it was not correct. What do you think about...

```ts
Boolean(multiline) || Boolean(autoGrowHeight)
```

This code is explicit, as it acknowledges that these values aren't proper booleans (but `boolean | undefined`), and it needs no linter silencing I think.